config/cmd/cmd.go

```diff
@@ -110,6 +113,11 @@ var (
 	DefaultCmd = newCmd()
 
 	DefaultFlags = []cli.Flag{
+		&cli.StringFlag{
+			Name:    "certificate_authorities",
+			EnvVars: []string{"MICRO_CERTIFICATE_AUTHORITIES"},
```

Comment:
MICRO_CLIENT_CA or MICRO_AUTH_CA might be better

Reply:
I know what you're saying however since the CA isn't just being used by the client, I think it's misleading. I'll merge as-is but we should come back and review all the changes once mTLS is done. I just didn't want to end up merging it as one huge PR akin to client-router.

Reply:
No one wants to set a value called MICRO_CERTIFICATE_AUTHORITIES for one thing, too long, another thing easy to misspell. From a complete usability standpoint its unusable so we should change it now to at the very least be MICRO_CA

Reply:
considering its probably used as either the RootCA or ClientCA field in tls certs then either MICRO_ROOT_CA or MICRO_CLIENT_CA might actually be more appropriate, they also use the CA acronym.